### DISCUSSION TITLE
سوال و جواب

### DISCUSSION THREAD

**Discussion by hayyaun** [hayyaun/kids#2] [Q&A]
# گروه رفع مشکلات

دوستای خوبم لطفا هر سوالی که دارید توی این گروه بفرستید
تا من یا بقیه دوستاتون ببینیم و رفع کنیم
هر سوال خوب یک ⭐ داره و هر جواب خوب هم همینطور
لطفا [صحبت های عمومی](https://github.com/hayyaun/kids/discussions/1) رو توی چت عمومی مطرح کنید

**Comment by AmirAliYeganeh** [comment]
سلام میشه لطفا امتیازات رو اعلام کنید؟

**Reply by hayyaun** [reply]
سلام امیرعلی جان امتیازات همیشه به روز هست. لینکش هم تو گروه درسیتون هم توی مخزن درسی اینجا هست.
https://github.com/hayyaun/kids/blob/main/SCORES.md
